src/controls/Drag.js

```diff
@@ -62,11 +64,14 @@ function DragControlMethod(element, pointerType, opts) {
 
   this._hammer.on("hammer.input", this._handleHammerEvent.bind(this));
 
-  this._hammer.on('panstart', this._handleStart.bind(this));
-  this._hammer.on('panmove', this._handleMove.bind(this));
-  this._hammer.on('panend', this._handleEnd.bind(this));
-  this._hammer.on('pancancel', this._handleEnd.bind(this));
+  if (!['pan', 'pinch'].includes(this._opts.hammerEvent)) {
```

Comment:
Please use `x != 'pan' && x != 'pinch'` here - `Array#includes` doesn't exist on IE11, which in principle we still support.